CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+Maintainers ([@open-telemetry/php-maintainers](https://github.com/orgs/open-telemetry/teams/php-maintainers)):
```

Comment:
I don't know if this link will be generated automatically, but at the moment is broken.

Reply:
This is a restricted URL for some reason; I think it's a permissions thing.